Mathlib/RingTheory/EuclideanDomain.lean

```diff
@@ -51,6 +51,13 @@ theorem isCoprime_div_gcd_div_gcd (hq : q ≠ 0) :
         (EuclideanDomain.mul_div_cancel' (gcd_ne_zero_of_right hq) <| gcd_dvd_right _ _).symm <|
       gcd_ne_zero_of_right hq
 
+theorem isCoprime_div_gcd_div_gcd' (hpq : GCDMonoid.gcd p q ≠ 0) :
```

Comment:
Why not use [isCoprime_div_gcd_div_gcd](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/EuclideanDomain.html#isCoprime_div_gcd_div_gcd) instead? The assumption `q ≠ 0` is easy to get from `gcd p q ≠ 0`.
(In any case, the primed version should have a docstring explaining how it differs from the unprimed one.)